### PR TITLE
Add Galaxy S4 T-Mobile [jfltetmo]

### PR DESCRIPTION
**Pull request by Qiangong2**

### CODE REVIEW

**Comment by lnjX** [comment]
- [x] Rename the repositories from `halium_*` to `android_*`
- [x] Use all already existing repositories from the Halium organisation
- [x] The remote should be called either `Qg2` or `Qiangong2`
- [x] The revision should be called either `halium-7.1` or `halium-5.1`
- [x] The revision should be added to the remote and omitted in each `<project>` tag
- [x] Don't fork all repositories, only those that you really needed to make modifications to!
- [x] Don't fork vendor repositories at all; remove all APKs and JARs from proprietary-files.txt in the device tree repos!

Also I'm not sure if there are real modifications made to the hardware on the `tmo` version; It probably makes more sense to make one generic port for all `jflte` versions (as `jfltexx`: https://github.com/Halium/projectmanagement/issues/59).

**Comment by Qiangong2** [comment]
Jfltetmo has slight differences, mostly with RIL. Also, when I've tried building it for jfltexx. It doesn't boot at all, it just hangs. If we can show that a single, unified tree would work just as well, then we can always merge them at a later time.

**Comment by lnjX** [comment]
Please do the listed changes. :smiley:

A unifed device tree is optional.

**Comment by Qiangong2** [comment]
@LNJ2 this should satisfy all of the requirements

File: manifests/samsung_jfltetmo.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+    <remote name="Qiangong2" fetch="https://github.com/Qiangong2/" revision="halium-7.1" />
+    <remote name="TheMuppets" fetch="https://github.com/TheMuppets/" revision="cm-14.1" />
+    <remote name="lineageos" fetch="https://github.com/lineageOS/" revision="cm-14.1" />
```
Comment (lnjX) [inline manifests/samsung_jfltetmo.xml]: TheMuppets and lineageos are not needed! Use `them` and `los` remotes.

File: manifests/samsung_jfltetmo.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    
+    <project path="device/samsung/jfltetmo" name="android_device_samsung_jfltetmo" remote="Qiangong2" />
+    <project path="kernel/samsung/jf" name="android_kernel_samsung_jf" remote="Qiangong2" />
+    <project path="device/samsung/jf-common" name="android_device_samsung_jf-common" remote="Qiangong2" />
+    <project path="device/qcom/common" name="android_device_qcom_common" remote="lineageos" />
+    <project path="external/stlport" name="android_external_stlport" remote="lineageos" />
+    <project path="device/samsung/qcom-common" name="android_device_samsung_qcom-common" remote="lineageos" />
+    <project path="hardware/samsung" name="android_hardware_samsung" remote="lineageos" />
+    <project path="vendor/samsung" name="proprietary_vendor_samsung" remote="TheMuppets" revision="cm-14.1" />
```
Comment (lnjX) [inline manifests/samsung_jfltetmo.xml]: Revision should be omitted here.

**Comment by lnjX** [comment]
Except the issues mentioned in the latest two comments it looks good, now!

**Comment by Qiangong2** [comment]
One more try @LNJ2 :D

**Pull request merged**